app/src/main/java/com/sopt/now/compose/presentation/login/LoginActivity.kt

```diff
@@ -0,0 +1,206 @@
+import androidx.compose.ui.unit.sp
+import com.sopt.now.compose.R
+import com.sopt.now.compose.presentation.User
+import com.sopt.now.compose.presentation.main.MainActivity
+import com.sopt.now.compose.presentation.signup.SignUpActivity
+import com.sopt.now.compose.ui.theme.NOWSOPTAndroidTheme
+import com.sopt.now.compose.util.getParcelable
+import com.sopt.now.compose.util.makeToastMessage
+
+class LoginActivity : ComponentActivity() {
+    private var user = User()
+
+    private lateinit var resultLauncher: ActivityResultLauncher<Intent>
+    override fun onCreate(savedInstanceState: Bundle?) {
+
+        super.onCreate(savedInstanceState)
+        setContent {
+            NOWSOPTAndroidTheme {
+                // A surface container using the 'background' color from the theme
+                Surface(
+                    modifier = Modifier.fillMaxSize(),
+                    color = MaterialTheme.colorScheme.background
+                ) {
+                    LoginScreen()
+                }
+            }
+        }
+        initResultLauncher()
+    }
+
+    @Composable
+    fun LoginScreen() {
+        var userId by remember { mutableStateOf("") }
+        var userPw by remember { mutableStateOf("") }
+
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(horizontal = 30.dp)
+        ) {
+            Spacer(
+                modifier = Modifier
+                    .height(30.dp)
+            )
+            Text(
+                text = stringResource(R.string.login_screen_title),
+                modifier = Modifier
+                    .align(Alignment.CenterHorizontally),
+                fontSize = 30.sp
+            )
+
+            Spacer(
+                modifier = Modifier
+                    .height(50.dp)
+            )
+            Column {
+                Text(stringResource(R.string.login_screen_id))
+                TextField(
+                    value = userId,
+                    onValueChange = { userId = it },
+                    placeholder = { Text(stringResource(R.string.login_screen_input_id)) },
+                    modifier = Modifier.fillMaxWidth()
+                )
+            }
+
+            Spacer(
+                modifier = Modifier
+                    .height(30.dp)
+            )
+            Column {
+                Text(stringResource(R.string.login_screen_pw))
+                TextField(
+                    keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Password),
+                    visualTransformation = PasswordVisualTransformation(),
+                    value = userPw,
+                    onValueChange = { userPw = it },
+                    placeholder = { Text(stringResource(R.string.login_screen_input_pw)) },
+                    modifier = Modifier.fillMaxWidth()
+                )
+            }
+
+            Spacer(
+                modifier = Modifier
+                    .height(300.dp)
```

Comment:
```suggestion
                    .weight(1f)
```
큰 dp 값을 사용할 필요 없이 weight를 사용하면 남은 공간을 전부 채워줍니다